packages/aws_common/lib/src/http/aws_http_operation.dart

```diff
@@ -43,16 +42,10 @@ class AWSHttpOperation<ResponseType extends AWSBaseHttpResponse>
     return result;
   }
 
-  /// The operation's progress sending the request, in bytes.
-  ///
-  /// If [cancel] is called before the request has been fully sent, the stream
-  /// will be closed and its `onDone` callback will fire.
+  @override
   final Stream<int> requestProgress;
```

Comment:
Does this need to be here still since it's provided in the mixin?  Is there any benefit to overriding it again here,  perhaps just for clarity?

Reply:
The mixin just adds the abstract getter to the class. It still needs to be implemented. In this case, it's implemented as a field rather than a getter.